src/vs-bicep/Bicep.VSLanguageServerClient.IntegrationTests/Utilities/VsHostUtility.cs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.IO;
+using Microsoft.Test.Apex.VisualStudio;
+using Microsoft.Test.Apex.VisualStudio.Shell;
+
+namespace Bicep.VSLanguageServerClient.IntegrationTests.Utilities
+{
+    public class VsHostUtility
+    {
+        public static VisualStudioHost? VsHost;
+
+        public static TextEditorDocumentWindowTestExtension? OpenFile(string fileRelativePath)
+        {
+            string filePath = Path.Combine(Path.GetDirectoryName(typeof(VsHostUtility).Assembly.Location), fileRelativePath);
+
+            if (VsHost is null)
+            {
+                return null;
```

Comment:
should this throw?  likewise down below

Reply:
Yes, updated here:
https://github.com/Azure/bicep/commit/b9b9d7136f73e212f867a5cd19818cc8b537b126